FEEDBACK.md

```diff
@@ -0,0 +1,334 @@
+# Bucket Feedback UI
+
+The Bucket SDK includes a UI you can use to collect feedback from user about particular features.
+
+## Global feedback configuration
+
+The Bucket SDK feedback UI is configured with reasonable defaults of an english language, positioning itself as a [dialog](#dialog) in the lower right-hand corner of the viewport and with a [light-mode theme](#custom-styling).
+
+These settings can be overwritten when initializing the Bucket SDK:
+
+```javascript
+bucket.init("bucket-tracking-key", {
+  feedback: {
+    ui: {
+      position: POSITION_CONFIG, // See positioning section
+      translations: TRANSLATION_KEYS, // See internationalization section
+
+      // Enable automated fedback collection. Default: `false`
+      automaticPrompting: boolean,
+
+      /**
+       * Do your own feedback prompt handling or override
+       * default settings at runtime.
+       */
+      promptHandler: (promptMessage, handlers) => {
+        // See automated feedback collection section
+      },
+    },
+  },
+});
+```
+
+See also:
+
+- [Positioning and behavior](#positioning-and-behavior) for the position option.
+- [Static language configuration](#static-language-configuration) if you want to translate the feedback UI.
+- [Automated feedback collection](#automated-feedback-collection) to override default configuration.
+
+## Manual feedback collection
+
+You can open up the feedback collection UI by calling `bucket.requestFeedback(options)` with the relevant options. This is great is you are interested in manually controlling the collection of feedback from your user, but want to use the convenience of the Bucket feedback UI to reduce the amount of code you need to maintain.
+
+Examples of this could be if you want to put a `give us feedback`-button on your page, or pop up the feedback UI at the end of specific user flows in your app.
```

Comment:
```suggestion
Examples of this could be if you want the click of a `give us feedback`-button or the end of a specific user flow, to trigger a pop-up displaying the feedback user interface.
```